.babelrc

```diff
@@ -1,5 +1,10 @@
 {
   "presets": [
     ["es2015", { "modules" : false }]
-  ]
+  ],
+  "env": {
+    "test": {
```

Comment:
hmm, it looks like there's no env stuff being set in the `package.json`. Is this being called correctly?

Reply:
Did some digging and there's no need to set  a babel `env` in `package.json` because Jest will automatically set the `env` to `test` if it's not provided (per the doc [here](https://github.com/facebook/jest/blob/master/docs/GettingStarted.md)).